src/components/Badge/Badge.module.scss

```diff
@@ -1,8 +1,8 @@
 .badge {
   position: relative;
   padding: 0 $padding-tiny;
-  text-transform: uppercase;
   font-size: 12px;
+  text-decoration: uppercase;
```

Comment:
Jak usunąłeś text-transform i dodałeś text-decoration, to te moje zmiany, które masz w linijce 111 nie będą działały teraz, czy się myle?

Reply:
oj powalilłem jedno z drugim, true, dzięki za czujność